test/e2e/utils/helpers.go

```diff
@@ -82,7 +82,7 @@ func IsNodeConfigDoneWithNodeTuningFunc(nodes []*corev1.Node) func(nc *scyllav1a
 }
 
 func RolloutTimeoutForScyllaCluster(sc *scyllav1.ScyllaCluster) time.Duration {
-	return syncTimeout + time.Duration(GetMemberCount(sc))*memberRolloutTimeout
+	return SyncTimeout + time.Duration(GetMemberCount(sc))*memberRolloutTimeout
```

Comment:
stale commit squash from different PR

Reply:
I think that was just the commits from the previous PR, it's intentionally exposed